native/src/widget/image.rs

```diff
@@ -30,24 +32,73 @@ pub fn viewer<Handle>(handle: Handle) -> Viewer<Handle> {
 ///
 /// <img src="https://github.com/iced-rs/iced/blob/9712b319bb7a32848001b96bd84977430f14b623/examples/resources/ferris.png?raw=true" width="300">
 #[derive(Debug, Hash)]
-pub struct Image<Handle> {
+pub struct Image<'a, Handle> {
```

Comment:
This fails to compile with `error[E0392]: parameter `'a` is never used` if the `a11y` feature isn't used.

So the struct needs a `PhantomData` member using this lifetime.